src/Tribe/Events/Events_Service_Provider.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tribe\Tickets\Events;
+
+class Events_Service_Provider extends \tad_DI52_ServiceProvider {
+
```

Comment:
:no_entry_sign: Missing doc comment for class Events_Service_Provider

<sub>hash: a1f4367fa962dfd639a5f53e1b68b1b5</sub>